app/helpers/course_helper.rb

```diff
@@ -28,4 +28,18 @@ def course_ucas_status(course)
       "not_running" => "Not running"
     }[course.attributes[:ucas_status]]
   end
+
+  def course_qualifactions(course)
+     # TODO: Should this be in the API? Remove this comment if this passes code review.
```

Comment:
I don't think the API should do sorting, by default, generally. There could be exceptions but I'm not sure if this qualifies?

Also, typo: `course_qualifactions` -> `course_qualifications`. :)

Reply:
Should the method be called "course_outcome"?

Reply:
Done